Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -1230,12 +1263,50 @@ static const std::map<TextureFormat, DecodingShaderInfo> s_decoding_shader_info{
         vec4 norm_color = GetPaletteColorNormalized(index);
         imageStore(output_image, ivec3(ivec2(coords), 0), norm_color);
       }
+      )"}},
+
+    // We do the inverse BT.601 conversion for YCbCr to RGB
+    // http://www.equasys.de/colorconversion.html#YCbCr-RGBColorFormatConversion
+    {TextureFormat::XFB,
+     {BUFFER_FORMAT_RGBA8_UINT, 0, 8, 8, false,
+      R"(
+      layout(local_size_x = 8, local_size_y = 8) in;
+
+      void main()
+      {
+        uvec2 uv = gl_GlobalInvocationID.xy;
+        int buffer_pos = int(u_src_offset + (uv.y * u_src_row_stride) + (uv.x / 2));
+        vec4 yuyv = texelFetch(s_input_buffer, buffer_pos);
+
+        // If our yuvyv value is equal to our 'out of range' key
```

Comment:
Can you please explain the logic behind using a texel with alpha=0? Why can't this just store vec4(0,0,0,255) and perform the final scanout without blending enabled?

Reply:
It's been a while since I looked at this, I can't recall all the details when I researched it.  Ultimately, this was part of phire's original PR and that's why I went with it.  The thinking is there are some XFB textures that need to show what is beneath them, they are "partial" XFBs.  That's why alpha blending is used over black colors.

But like I said in my other comment, as far as we know this only affects one game and our implementation likely isn't 100% correct for that game since XFB2Tex doesn't work (only XFB2Ram).

If you want, we can rip this whole partial-xfb support out.  We'd want to ask @JMC47 to see how he'd feel about not having this capability though.